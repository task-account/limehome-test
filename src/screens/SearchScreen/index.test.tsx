import renderer from "react-test-renderer";
import { SearchScreen } from ".";

jest.mock("@utils/hooks", () => ({
  withStyles: () => () => ({ useStyles: jest.fn() })
}));

describe(`SearchScreen`, () => {
  it("render", async () => {
    const tree = renderer.create(<SearchScreen />).toJSON();
    await new Promise(setImmediate);
    expect(tree).toMatchSnapshot();
  });
});
