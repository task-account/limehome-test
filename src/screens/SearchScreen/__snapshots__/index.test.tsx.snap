// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchScreen render 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
>
  <View>
    <TextInput
      onChangeText={[Function]}
      placeholder="Type name"
      value=""
    />
    <RCTScrollView
      ItemSeparatorComponent={[Function]}
      ListEmptyComponent={
        <View>
          <Spacer
            vertical={100}
          />
          <Text>
            Nothing was found
          </Text>
        </View>
      }
      ListFooterComponent={<View />}
      data={Array []}
      getItem={[Function]}
      getItemCount={[Function]}
      keyExtractor={[Function]}
      onContentSizeChange={[Function]}
      onLayout={[Function]}
      onMomentumScrollBegin={[Function]}
      onMomentumScrollEnd={[Function]}
      onScroll={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      removeClippedSubviews={false}
      renderItem={[Function]}
      scrollEventThrottle={50}
      showsVerticalScrollIndicator={false}
      stickyHeaderIndices={Array []}
      viewabilityConfigCallbackPairs={Array []}
    >
      <View>
        <View
          onLayout={[Function]}
        >
          <View
            style={
              Object {
                "flex": undefined,
                "marginBottom": undefined,
                "marginHorizontal": undefined,
                "marginLeft": undefined,
                "marginRight": undefined,
                "marginTop": undefined,
                "marginVertical": 100,
              }
            }
          />
          <Text>
            Nothing was found
          </Text>
        </View>
        <View
          onLayout={[Function]}
        >
          <View />
        </View>
      </View>
    </RCTScrollView>
  </View>
</RCTSafeAreaView>
`;
